packages/server/graphql/public/types/StopRecurrenceSuccess.ts

```diff
@@ -1,13 +1,14 @@
-import MeetingTeamPrompt from '../../../database/types/MeetingTeamPrompt'
+import MeetingSeriesId from '../../../../client/shared/gqlIds/MeetingSeriesId'
 import {StopRecurrenceSuccessResolvers} from '../resolverTypes'
 
 export type StopRecurrenceSuccessSource = {
-  meetingId: string
+  meetingSeriesId: string
 }
 
 const StopRecurrenceSuccess: StopRecurrenceSuccessResolvers = {
-  meeting: async ({meetingId}, _args, {dataLoader}) => {
-    return dataLoader.get('newMeetings').load(meetingId) as Promise<MeetingTeamPrompt>
+  meetingSeries: async ({meetingSeriesId}, _args, {dataLoader}) => {
+    const id = MeetingSeriesId.split(meetingSeriesId)
```

Comment:
+1 I don't know what you've discussed before with regards to GraphQL IDs, but I personally would return database id's in the source objects from the mutations and do the ID conversion in the GraphQLObject resolvers as I perceive these as a conversion layer.

Reply:
very true, good catch!